process/block/displayBlock.go

```diff
@@ -106,10 +120,10 @@ func createDisplayableShardHeaderAndBlockBody(
 	return tableHeader, shardLines
 }
 
-func displayTxBlockBody(lines []*display.LineData, body block.Body) []*display.LineData {
-	txCounterMutex.Lock()
-	txsCurrentBlockProcessed = 0
-	txCounterMutex.Unlock()
+func displayTxBlockBody(lines []*display.LineData, body block.Body, txCounter *transactionCounter) []*display.LineData {
+	txCounter.mutex.Lock()
+	txCounter.currentBlockTxs = 0
```

Comment:
Use local variables for this. There is no need for the locks and unlocks in the for. One lock and one unlock after the for should be enough.

Reply:
Done.